src/js/disability-benefits/utils/helpers.js

```diff
@@ -199,3 +202,27 @@ export function isClaimComplete(claim) {
 export function itemsNeedingAttentionFromVet(events) {
   return events.filter(event => event.status === 'NEEDED' && event.type === 'still_need_from_you_list').length;
 }
+
+export function makeRequest(url, userOptions = {}) {
```

Comment:
That's what I wanted to do initially. It doesn't really work as part of the promise chain because the auth error should stop the promise from executing the success path and the normal error path. Promises only have two paths, so you would need to pass an error callback or something, which is confusing alongside promises. Plus, you need to pass `dispatch` in as well, though that's less of an issue.

It may be worth it to ditch promises here and just use callbacks.